core/embed/rust/src/micropython/obj.rs

```diff
@@ -256,6 +258,14 @@ impl TryFrom<&[u8]> for Obj {
     }
 }
 
+impl TryFrom<()> for Obj {
+    type Error = Error;
+
+    fn try_from(_val: ()) -> Result<Self, Self::Error> {
+        catch_exception(Self::const_none)
```

Comment:
`const_none` currently doesn't throw exceptions as it doesn't call micropython code, and I think it's reasonable to assume that it will continue in the future.